### PR TITLE
Adds indicator of a player-visible voting log

### PR DESCRIPTION
**Pull request by langford**
This makes the existence of a voting log front and center to all players when it exists. This lets a new player notice what other players are working off of, and it enables a group to maintain a limited log by reminding the ST to manually clear a log. 

This has used and complied with the lint that was shown in npm run serve. Happy to run another if that's not the right one. 

Screen Shot 2021-03-08 at 5 46 19 PM

### CODE REVIEW

File: src/components/Menu.vue [hunk 1]
```diff
@@ -1,5 +1,18 @@
 <template>
   <div id="controls">
+    <span
+      class="nomlogSummary"
```
Comment (bra1n) [inline src/components/Menu.vue:4]: Please use kebab-case for CSS class names. :smiley:

File: src/components/Menu.vue [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+      v-show="session.voteHistory.length"
```
Comment (bra1n) [inline src/components/Menu.vue:5]: It should probably also only be visible when the player is actually in a live session, so I'd suggest checking for `session.sessionId` as well.

File: src/components/Menu.vue [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
+      @click="toggleModal('voteHistory')"
+      :title="
+        `${session.voteHistory.length} recent ${
+          session.voteHistory.length == 1 ? 'nomination' : 'nominations'
```
Comment (bra1n) [inline src/components/Menu.vue:9]:
```suggestion
        `${session.voteHistory.length} recent 
          nomination${session.voteHistory.length > 1 ? 's' : ''
```

File: src/components/Menu.vue [hunk 4]
```diff
@@ -366,6 +379,12 @@ export default {
     margin-left: 10px;
   }
 
+  span.nomlogSummary {
+    color: $townsfolk;
+    margin-top: 7px;
+    margin-left: 10px;
```
Comment (bra1n) [inline src/components/Menu.vue:385]: Aren't those already inherited from [line 378](https://github.com/bra1n/townsquare/pull/125/files#diff-e065371e6591864d797ae6468724bc4bcb3d329b950921fdf19f4c18e07dc500R378-R379)?

**Review by bra1n — CHANGES_REQUESTED** [review]

File: src/components/Menu.vue (hunk 3 above)
Comment (langford) [reply src/components/Menu.vue:9]: I'd prefer to not change this: In English, singular case is only used for 1 item, plural is used for 0 as well. That doesn't matter *right now*, but it does make changing policy about this item (and translation of the app) easier.

File: src/components/Menu.vue (hunk 1 above)
Comment (langford) [reply src/components/Menu.vue]: willdo

File: src/components/Menu.vue (hunk 4 above)
Comment (langford) [reply src/components/Menu.vue:385]: Seems like they are. I was originally looking at different separations/placements and didn't remove them. I will remove them.

File: src/components/Menu.vue (hunk 3 above)
Comment (bra1n) [reply src/components/Menu.vue:9]: Well, given that the icon is only visible for voteHistory.length >= 1, using the plural for 0 would be moot, but I see your point. 👍

**Review by bra1n — APPROVED** [review]
Tentatively approved. Need to run it locally once before merging first, and an addition to the CHANGELOG.md would also be welcome. 😉

**Comment by langford** [comment]
> Tentatively approved. Need to run it locally once before merging first, and an addition to the CHANGELOG.md would also be welcome. 😉

Added a line. 

Btw, when you run the local running test: The nomination log is *not* resynced when a player hard reloads a page. This data situation pre-existed my issue. #136 for more details.

**Comment by bra1n** [comment]
> > Tentatively approved. Need to run it locally once before merging first, and an addition to the CHANGELOG.md would also be welcome. 😉
> 
> Added a line.
> 
> Btw, when you run the local running test: The nomination log is _not_ resynced when a player hard reloads a page. This data situation pre-existed my issue. #136 for more details.

That's intentional. Thanks for the changelog! :-)

**Pull request merged**